doc/interoperability/exceptions.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: doc
+title: Handling Exceptions 
+---
+
+Scala.js can both throw and catch exceptions as you normally would in scala. Scala.js also supports exceptions originating in or being caught by JS.
+
+Scala.js' wrapper, `js.JavasScriptException`, wraps any JS value although it typically wraps a `js.Error`. The wrapper indicates that  special "wrapper" processing should occur during interoperation with JS.
```

Comment:
I don't think this is a correct way to describe it. As is, it feels like what I call a "compiler feature" (rooted in implementation details), rather than a "language feature" (rooted in semantics).

I would suggest something like this instead:

> Both Scala and JavaScript feature exception handling, but Scala only allows instances of `Throwable` to be thrown/caught (which is enforced by the type checker), whereas JavaScript can throw/catch any type of value.
> To reconcile the two worlds, Scala.js lifts all exceptions that are not instances of `Throwable` inside `js.JavaScriptException`.
> This lifting works both ways: